e2e_test.go

```diff
@@ -27,6 +27,9 @@ func TestE2e(t *testing.T) {
 			t.Logf("Skipping operator install as requested")
 		}
 	})
+	if t.Failed() {
+		return
```

Comment:
You can also do `t.Fatal()` to fail the test and log a warning.

Reply:
Actually - using `t.Fatal()` here is probably negligible since we're already returning from the outer most layer of the test suite (I was thinking we were running inside another nested test).